run/pdf2john.py

```diff
@@ -8,13 +8,14 @@
 
 import argparse
 import logging
+import sys
 
 try:
     from pyhanko.pdf_utils.misc import PdfReadError
     from pyhanko.pdf_utils.reader import PdfFileReader
 except ImportError:
-    print("pyhanko is missing, run 'pip install --user pyhanko==0.20.1' to install it!")
-    exit(1)
+    print("pyhanko is missing, run 'pip install --user pyhanko' to install it!", file=sys.stderr)
+    sys.exit(1)
```

Comment:
BTW, we also have uses of plain `exit` in a few other scripts:

```
apop2john.py:    exit(1)
apop2john.py:    exit(1)
authenticator2john.py:    exit(1)
authenticator2john.py:        exit(1)
bestcryptve2john.py:        exit(0)
bitlocker2john.py:            exit()
kdcdump2john.py:    exit()
mosquitto2john.py:            exit(1)
```

Reply:
I could fix them. But these are rather minor linting issues and many scripts produce several linting issues. Too many to go over everything in every script. So tackling exactly the `exit()` issue would feel a bit random and not very important.